package.json

```diff
@@ -1,16 +1,17 @@
 {
   "name": "discord-api-docs",
-  "version": "1.1.1",
+  "version": "1.1.2",
   "description": "Documentation for using Discord's API",
   "main": "index.js",
   "directories": {
     "doc": "docs"
+    "discord": "discord"
```

Comment:
There is no discord directory at the repository root

Reply:
Yes i found that out, that's why i took it out

Reply:
I was looking at a wrong diff/commit in the PR; I did not see you removed it in a later commit.